index.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
should we remove for now?

Reply:
I'd like to keep it since I link to it from the package.json